chpl/chpl-service/src/main/java/gov/healthit/chpl/certificationId/Validator2014.java

```diff
@@ -147,6 +149,13 @@ protected boolean isInpatientCqmsValid() {
                 ++cqmCount;
             }
         }
+        
+        if(!(this.counts.get("cqmsInpatientRequiredMet") >= this.counts.get("cqmsInpatientRequired"))){
```

Comment:
I know checkstyle doesn't like `if(` but prefers `if (`

Reply:
Wouldn't this be equivalent?
`if (this.counts.get("cqmsInpatientRequiredMet") < this.counts.get("cqmsInpatientRequired")) {`